code/game/jobs/job/security.dm

```diff
@@ -208,6 +209,7 @@
 	jobtype = /datum/job/officer
 
 	uniform = /obj/item/clothing/under/rank/security
+	suit = /obj/item/clothing/suit/storage/hazardvest/security/officer
```

Comment:
all this is going to do is fuck up loadout preview, put it in the locker if anything

Reply:
this is intentional because every other security job has a suit clothing article, did not want to have an outlier